### ISSUE TITLE
Cannot create alias of alias - Mordred fails.

### ISSUE THREAD

**Issue by lukaszgryglicki** [chaoss/grimoirelab-sirmordred#241]
When I configured my ElasticSearch instance to use aliases it causes Mordred to fail.

My `/git` is an alias that points to `git_v1` same with all other data sources.
So everyone like `jira_raw` is really an alias to `jira_raw_v1`.
This is because I want to be able to easily change what `git`, `jira`, `github` and others mean.
It was Ok for Perceval and OK for ELK but Mordred tries to create `alias of alias` and fails, the exact error message is:
```
2018-10-18 13:06:49,670 - sirmordred.task_manager - ERROR - Exception in Task Manager 400 Client Error: Bad Request for url: http://localhost:9200/_aliases
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_manager.py", line 92, in run
    task.execute()
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_enrich.py", line 308, in execute
    raise e
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_enrich.py", line 298, in execute
    self.__enrich_items()
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_enrich.py", line 181, in __enrich_items
    task_aliases.execute()
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_panels.py", line 534, in execute
    self.__create_aliases()
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_panels.py", line 520, in __create_aliases
    self.__create_alias(es_col_url, index_raw, real_alias + "-raw")
  File "/usr/local/lib/python3.6/dist-packages/sirmordred/task_panels.py", line 495, in __create_alias
    res.raise_for_status()
  File "/usr/lib/python3/dist-packages/requests/models.py", line 935, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost:9200/_aliases
```

I'll try to dig Python code to fix that and then eventually create a PR.

**Comment by lukaszgryglicki** [comment]
OK I know how to fix, it works locally, creating PR...

**Comment by lukaszgryglicki** [comment]
Created PR that fixes this: #242

**Comment by valeriocos** [comment]
Aliases can be set up with the aliases.json: https://github.com/chaoss/grimoirelab-sirmordred/blob/master/aliases.json.
